app/src/test/kotlin/com/wire/android/mapper/UIParticipantMapperTest.kt

```diff
@@ -51,7 +51,8 @@ class UIParticipantMapperTest {
         memberDetails.userId == uiParticipant.id
                 && memberDetails.name == uiParticipant.name
                 && memberDetails.handle == uiParticipant.handle
-                && memberDetails.avatar(wireSessionImageLoader) == uiParticipant.avatarData
+                && TODO()
+                // && memberDetails.avatar(wireSessionImageLoader) == uiParticipant.avatarData
```

Comment:
Can't we just compare `memberDetails.user.avatar(wireSessionImageLoader) == uiParticipant.avatarData`? 🤔